(158,) [ 0.    0.05  0.1   0.15  0.2   0.25  0.3   0.35  0.4   0.45  0.5   0.55
  0.6   0.65  0.7   0.75  0.8   0.85  0.9   0.95  1.    1.05  1.1   1.15
  1.2   1.25  1.3   1.35  1.4   1.45  1.5   1.55  1.6   1.65  1.7   1.75
  1.8   1.85  1.9   1.95  2.    2.05  2.1   2.15  2.2   2.25  2.3   2.35
  2.4   2.45  2.5   2.55  2.6   2.65  2.7   2.75  2.8   2.85  2.9   2.95
  3.    3.05  3.1   3.15  3.2   3.25  3.3   3.35  3.4   3.45  3.5   3.55
  3.6   3.65  3.7   3.75  3.8   3.9   4.    4.1   4.2   4.3   4.4   4.5
  4.6   4.7   4.8   4.9   5.    5.1   5.2   5.3   5.4   5.5   5.6   5.7
  5.8   5.9   6.    6.1   6.2   6.3   6.4   6.5   6.6   6.7   6.8   6.9
  7.    7.1   7.2   7.3   7.4   7.5   7.6   7.7   7.8   7.9   8.    8.1
  8.2   8.3   8.4   8.5   8.6   8.7   8.8   8.9   9.    9.1   9.2   9.3
  9.4   9.5   9.6   9.7   9.95 10.2  10.45 10.7  10.95 11.2  11.45 11.7
 11.95 12.2  12.45 12.7  12.95 13.2  13.45 13.7  13.95 14.2  14.45 14.7
 14.95 15.2 ]
(24,) [  0.  15.  30.  45.  60.  75.  90. 105. 120. 135. 150. 165. 180. 195.
 210. 225. 240. 255. 270. 285. 300. 315. 330. 345.] deg
3792
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/u21/kianmilani/.conda/envs/poppy39-exp-env/lib/python3.9/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/home/u21/kianmilani/.conda/envs/poppy39-exp-env/lib/python3.9/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/home/u21/kianmilani/.local/lib/python3.9/site-packages/PyPROPER3-3.2.5-py3.9-linux-x86_64.egg/proper/prop_execute_multi.py", line 38, in prop_execute_multi
    psf, sampling = proper.prop_run(routine_name, lamda, gridsize, PASSVALUE = passvalue, QUIET = quiet, PHASE_OFFSET = phase_offset, IS_MULTI=True)
  File "/home/u21/kianmilani/.local/lib/python3.9/site-packages/PyPROPER3-3.2.5-py3.9-linux-x86_64.egg/proper/prop_run.py", line 145, in prop_run
    psf_ilam, pixscale = func(lam[ilam], gridsize, kwargs["PASSVALUE"])
  File "/home/u21/kianmilani/Documents/disk-processing/roman_phasec.py", line 273, in roman_phasec
    raise Exception(' ')
Exception:  
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/u21/kianmilani/Documents/disk-processing/make_hlc_band1_psfs.py", line 98, in <module>
    (wfs, pxscls_m) = proper.prop_run_multi('roman_phasec', lam_array, npsf, QUIET=True, PASSVALUE=options)
  File "/home/u21/kianmilani/.local/lib/python3.9/site-packages/PyPROPER3-3.2.5-py3.9-linux-x86_64.egg/proper/prop_run_multi.py", line 146, in prop_run_multi
    res = pool.map(proper.prop_execute_multi, params)
  File "/home/u21/kianmilani/.conda/envs/poppy39-exp-env/lib/python3.9/multiprocessing/pool.py", line 364, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/home/u21/kianmilani/.conda/envs/poppy39-exp-env/lib/python3.9/multiprocessing/pool.py", line 771, in get
    raise self._value
Exception:  
Error in roman_phasec: requested wavelength not within 0.1 nm of nearest available FPM wavelength.
  requested (um) = 0.5821875  closest available (um) = 0.5844875
Error in roman_phasec: requested wavelength not within 0.1 nm of nearest available FPM wavelength.
  requested (um) = 0.5965625  closest available (um) = 0.598752
Error in roman_phasec: requested wavelength not within 0.1 nm of nearest available FPM wavelength.
  requested (um) = 0.5678124999999999  closest available (um) = 0.56573
Error in roman_phasec: requested wavelength not within 0.1 nm of nearest available FPM wavelength.
  requested (um) = 0.5534374999999999  closest available (um) = 0.551135
Detailed performance metrics for this job will be available at https://metrics.hpc.arizona.edu/#job_viewer?action=show&realm=SUPREMM&resource_id=73&local_job_id=4162402 by 8am on 2022/08/20.
